distributed/node.py

```diff
@@ -94,6 +94,21 @@ def _setup_logging(self, logger: logging.Logger) -> None:
         logger.addHandler(self._deque_handler)
         weakref.finalize(self, logger.removeHandler, self._deque_handler)
 
+        loggers_to_register = dask.config.get("distributed.admin.log-events")
```

Comment:
I think it would be good to have another test that checks that the configuration works as expected (e.g., changing the configuration will result in some log being propagated or not).